src/cli/fleet.toit

```diff
@@ -471,6 +472,11 @@ class Fleet:
     new-file := fleet-file_.with --recovery-urls=[]
     new-file.write
 
+  recovery-info -> ByteArray:
+    broker.server-config.fill-certificate-ders: certificate-roots.MAP[it].raw
+    json-config := broker.server-config.to-json --der-serializer=: base64.encode it
```

Comment:
This is not the format the device expects.

Reply:
done.